.github/workflows/catpowder.yml

```diff
@@ -46,6 +46,24 @@ jobs:
         echo -e "\tPasswordAuthentication no" >> $HOME/.ssh/config
         echo -e "\tUser ${{ secrets.USERNAME }}" >> $HOME/.ssh/config
         echo -e "\tPort ${{ secrets.PORTNUM }}" >> $HOME/.ssh/config
+    - name: Bootstrap VMs
+      shell: bash
+      run: |
+        echo "Running iptables rule on SERVER"
+        ssh $SERVER "sudo iptables -C OUTPUT -p tcp -o eth1 --tcp-flags RST RST -j DROP || \
+                     sudo iptables -A OUTPUT -p tcp -o eth1 --tcp-flags RST RST -j DROP"
+
+        echo "Running ip addr flush on SERVER"
+        ssh $SERVER "sudo ip -br addr show dev eth1 | grep -q ' UP ' && \
+                     sudo ip addr flush dev eth1"
+
+        echo "Running iptables rule on CLIENT"
+        ssh $CLIENT "sudo iptables -C OUTPUT -p tcp -o eth1 --tcp-flags RST RST -j DROP || \
+                     sudo iptables -A OUTPUT -p tcp -o eth1 --tcp-flags RST RST -j DROP"
+
+        echo "Running ip addr flush on CLIENT"
+        ssh $CLIENT "sudo ip -br addr show dev eth1 | grep -q ' UP ' && \
+                     sudo ip addr flush dev eth1"
```

Comment:
We may want to add these steps in the release pipeline as well. A retrigger of the pipeline for any reason, would not re-run the bootstrap steps.

Reply:
They are already added in the release-pipeline section as well. (If you scroll to Ln no. 117)